### PR TITLE
Fix Response Types

### PR DESCRIPTION
**Pull request by affan-iconsult**

### CODE REVIEW

File: user-service/src/main/java/com/iconsult/userservice/config/KafktaConfig.java [hunk 1]
```diff
@@ -6,7 +6,7 @@
 import org.springframework.kafka.config.TopicBuilder;
 
 @Configuration
-public class KakftaConfig {
+public class KafktaConfig {
```
Comment (Iamsiraj) [inline user-service/src/main/java/com/iconsult/userservice/config/KafktaConfig.java]: kafka

File: user-service/src/main/java/com/iconsult/userservice/config/KafktaConfig.java [hunk 2]
```diff
@@ -6,7 +6,7 @@
...
     @Bean
     public NewTopic topic1()
```
Comment (Iamsiraj) [inline user-service/src/main/java/com/iconsult/userservice/config/KafktaConfig.java]: make it more understandable , if this is OTP then make it topicOTP

**Review by Iamsiraj — CHANGES_REQUESTED** [review]

File: user-service/src/main/java/com/iconsult/userservice/config/KafktaConfig.java (hunk 1 above)
Comment (affan-iconsult) [reply user-service/src/main/java/com/iconsult/userservice/config/KafktaConfig.java]: Done

File: user-service/src/main/java/com/iconsult/userservice/config/KafktaConfig.java (hunk 2 above)
Comment (affan-iconsult) [reply user-service/src/main/java/com/iconsult/userservice/config/KafktaConfig.java]: Done

**Review by Iamsiraj — APPROVED** [review]

**Pull request merged**